views/layouts/authenticated-layout.pug

```diff
@@ -41,16 +42,22 @@ html
                   .dropdown__item--label#weekly-summary Weekly Summary
               li.dropdown__item
                 a.dropdown__item--link(href='/accounts/logout')
+              li.dropdown__item
+                a.dropdown__item--export-link Export All
+              li.dropdown__item
+                a.dropdown__item--link(href='/accounts/logout')
                   .dropdown__item--label Log Out
               li.dropdown__item.dropdown__item--header= user.email
 
     .container
       block content
-        
+
     include ../accounts/weekly-report.pug
 
     script(src='https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js')
     script(src='/javascripts/fetch-params.js')
     script(src='/javascripts/items.js')
     script(src='/javascripts/starred.js')
     script(src='/javascripts/weekly-review.js')
+    script(src='/javascripts/client.js')
+    script(src='/javascripts/client.js')
```

Comment:
just wondering why it's added here twice 

+    script(src='/javascripts/client.js')
 +    script(src='/javascripts/client.js')